sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -87,27 +97,45 @@ describe('<item-list>', () => {
 
   it('displays a row per item in list', () => {
     const children = testFixture.$.listContainer.querySelectorAll('paper-item');
-    assert(children.length === 3, 'three rows should appear');
+    assert(children.length === 5, 'five rows should appear');
+  });
+
+  it('starts out with all items unselected', () => {
+    assert(!isSelected(0) && !isSelected(1) && !isSelected(2), 'all items should be unselected');
```

Comment:
Now that you have five items, this test is not testing what it says it is. :-)
Probably would be better to assert that selectedIndices == []